package.json

```diff
@@ -27,11 +24,10 @@
   },
   "homepage": "https://github.com/brave/autoplay-whitelist#readme",
   "dependencies": {
-    "cppunitlite": "^1.0.0",
-    "hashset-cpp": "^2.1.0",
-    "mkdirp": "^0.5.1",
-    "nan": "^2.10.0",
     "node-gyp": "^3.8.0"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "hashset-cpp": "^2.1.0",
```

Comment:
I think hashset-cpp should be in `dependencies` and not `devDependencies`

Reply:
done